Sources/SwiftDocC/Infrastructure/Workspace/DocumentationBundle+Info.swift

```diff
@@ -91,20 +108,23 @@ extension DocumentationBundle {
         ///   - defaultCodeListingLanguage: The default language identifier for code listings in the bundle.
         ///   - defaultAvailability: The default availability for the various modules in the bundle.
         ///   - defaultModuleKind: The default kind for the various modules in the bundle.
+        ///   -  inheritDefaultAvailability: The option to enable or disable symbol availability inheritance from the module default availability.
```

Comment:
nit: alignment
```suggestion
        ///   - inheritDefaultAvailability: The option to enable or disable symbol availability inheritance from the module default availability.
```

Reply:
Resolved in https://github.com/swiftlang/swift-docc/pull/1024/commits/6055aa94f29b2e827aafa8598f531adbeccb25e7